### ISSUE TITLE
Timeline doesn't work since august monthly update

### ISSUE THREAD

**Issue by viing937** [RikkaApps/Riru-LocationReportEnabler#21]
Timeline doesn't work on my pixel 3 with ***Android P*** since august monthly update.
The timeline data before this update is still available in maps app.
Also, I can't see weathers on "At a glance" widget.

Google feed still works.

Magisk and modules are all latest, and no other modules installed.

Magisk: v19.3
Riru-Core: v19.4
Riru-Location Report Enabler: v9.2 
Riru-Storage Redirect(Enhanced mode): v19.6

**Comment by viing937** [comment]
Weathers on "At a glance" widget sometimes appear, but the timeline still not work.

**Comment by RikkaW** [comment]
`adb logcat -s LocationReportEnabler`

If there are logs like below, the module works.
```
I/LocationReportEnabler: install hook for 0:com.google.android.gms
I/LocationReportEnabler: android::base::GetProperty: gsm.sim.operator.numeric -> 310030
I/LocationReportEnabler: android::base::GetProperty: gsm.sim.operator.numeric -> 310030
I/LocationReportEnabler: install hook for 0:com.google.android.gsf
I/LocationReportEnabler: android::base::GetProperty: gsm.sim.operator.numeric -> 310030
```

**Comment by viing937** [comment]
I see a lot of `310030` logs, it seems the module works.
Maybe Google change check logic to enable the timeline?
Does anyone have the same problem since August monthly update?

`
I/LocationReportEnabler: android::base::GetProperty: gsm.sim.operator.numeric -> 310030
`

**Comment by viing937** [comment]
Hi, I found the problem. 
In the Google Location History settings, it shows *on*.
However, pay attention to the **Devices on this account** section below and expand it, there are two identical devices, and the device marked as "this device" is not selected.
It seemed as if it had registered my phone twice as 2 separate devices after I do the factory flash for August update.
I turned this device on, and my location history started working again.
I searched the Internet, but I can't find any way to delete the ghost device. Hopes that time can solve this problem.
